index.html

```diff
@@ -64,207 +71,244 @@ <h2 class="media-h2">LET'S CONNECT</h2>
         </ul>
     </section>
     
-    
     <section class="works">
-        <div class="works-container-tonic1">
-            <div class="item-tonic1">
-                <div class="card-tonic1">
-                    <div class="card-body-tonic1">
-                        <img class="card-image-tonic1" src="./images/works1.png" alt="works1">
-                        <h3 class="card-title-tonic1">Tonic</h3>
-                        <div class="canopy-frm-tonic1">
-                            <div class="client-tonic1">
-                                <h3>CANOPY</h3>
+        
+        <div class="works-container">
+            
+            <div class="item">                
+                <div class="card">                    
+                    <div class="card-body">
+                        <div class="card-image-tonic1"> 
+                            <img class="card-image1-tonic1 img-right" src="./images/desk-works1.png" alt="works1">
```

Comment:
- [ ] [OPTIONAL] Everything is working well but there was a typo. Please kindly change  `card-image1-tonic1` to `card-image-tonic1`, I think you added an additional `1` by mistake 👍

Reply:
Fixed,
Thank you.